nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -1337,13 +1337,6 @@ CREATE ROLE postgres LOGIN SUPERUSER;
      </itemizedlist>
     </para>
    </listitem>
-   <listitem>
-    <para>
-      The GNOME desktop manager no longer default installs <package>gnome3.epiphany</package>.
-      It was chosen to do this as it has a usability breaking issue (see issue <link xlink:href="https://github.com/NixOS/nixpkgs/issues/98819">#98819</link>)
-      that makes it unsuitable to be a default app.
-    </para>
-   </listitem>
```

Comment:
\we should not retroactively alter 20.09 release notes. We could add an item that says it changed back in Okapi though.

Reply:
Well, I think it should be backported to 20.09, since it was done because of the webkitgtk issue. See #107981. If we want to preserve the release note history, we could add an additional item redacting the previous one, though.

Reply:
let's not remove this rl-2009. let's edit for the Okapi note to say it was readded. and then add a note below this saying that x & x change allowed it to be readded to defaults.